vendor/engines/bulk_email/app/controllers/bulk_email/bulk_email_controller.rb

```diff
@@ -21,15 +21,12 @@ class BulkEmailController < ApplicationController
     def search
       @searcher = BulkEmailSearcher.new(@search_fields)
       @users = @searcher.do_search
+    end
 
-      respond_to do |format|
-        format.html
-
-        format.csv do
-          filename = "bulk_email_#{params[:bulk_email][:user_types].join('-')}.csv"
-          set_csv_headers(filename)
-        end
-      end
+    def download
+      @users = User.where_ids_in(params[:recipient_ids])
```

Comment:
Yeah, we would need to hang on to more of the search criteria because otherwise users don't really "belong" to a facility. I'm wondering: does this open anything new to a user who also has access to search `/facilities/:id/users`?